src/deploy/table_report.sql

```diff
@@ -0,0 +1,28 @@
+-- Deploy maevsi:table_report to pg
+-- requires: schema_public
+-- requires: table_account_public
+-- requires: table_event
+-- requires: table_upload
+
+BEGIN;
+
+CREATE TABLE maevsi.report (
+  id                  UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+  author_account_id   UUID NOT NULL REFERENCES maevsi.account(id),
+  reason              TEXT NOT NULL CHECK (char_length("reason") > 0 AND char_length("reason") < 2000),
+  target_account_id   UUID REFERENCES maevsi.account(id),
+  target_event_id     UUID REFERENCES maevsi.event(id),
+  target_upload_id    UUID REFERENCES maevsi.upload(id),
+  CHECK (num_nonnulls(target_account_id, target_event_id, target_upload_id) = 1),
+  UNIQUE (author_account_id, target_account_id, target_event_id, target_upload_id)
+);
```

Comment:
I recommend
- adding a column "created" with the timestamp when the report record is inserted.
- adding a column "report_category" with an enum type indicating what kind of report is submitted (e.g. 'offensive content', ..., 'others'), check how this is done in other social media apps.

Reply:
@Bobomify shall we already implement a hardcoded set of reasons or just allow a "free text input" for now? Like, is there a business reason to offer a *limited* set of reasons? I can tell you in our next meeting what I think of by this question.

Reply:
@sthelemann thank you for implementing your first suggestion so speedily! :raised_hands: